packages/core/src/internals/files.ts

```diff
@@ -97,26 +97,26 @@ function readFirstOf<
   };
 }
 
-function isDirectory(source: string): boolean {
+export function isDirectory(source: string): boolean {
   return fs.lstatSync(source).isDirectory();
 }
 
-function isFile(source: string): boolean {
+export function isFile(source: string): boolean {
   return fs.lstatSync(source).isFile();
 }
 
-function readDirectory(source: string): string[] {
+export function readDirectory(source: string): string[] {
   return fs.readdirSync(source, { encoding: _format });
 }
 
-function mkdir(
+export function mkdir(
   target: string,
   options: { recursive: boolean }
 ): string | undefined {
   return fs.mkdirSync(target, options);
 }
 
-function exists(pathToFile: string): boolean {
+export function exists(pathToFile: string): boolean {
```

Comment:
is this better than `fs.extists`?

Reply:
If you need the catch, yes! I'm not sure yet we need this file at all

Reply:
fs.existsSync() is just true or false, not much else to it.